purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -614,6 +614,58 @@ class Purchases @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE) intern
         } ?: retrievePurchaseInfo(identityManager.currentAppUserID, listener)
     }
 
+    /**
+     * This function will change the current appUserID.
+     * Typically this would be used after a log out to identify a new user without calling configure
+     * @param newAppUserID The new appUserID that should be linked to the currently user
+     * @param [listener] An optional listener to listen for successes or errors.
+     */
+    @JvmOverloads
+    internal fun logIn(
```

Comment:
Let's add `@JvmSynthetic` too so that it doesn't show up in Java apps. Otherwise this function can be called from Java doing:

`Purchases.getSharedInstance().logIn$purchases_debug`